.circleci/config.yml

```
@@ -38,7 +38,7 @@ jobs:
       - checkout
       - setup_remote_docker
       - pack/install-pack:
-          version: 0.23.0
+          version: 0.24.1
```

Pack 0.26.0 is now out:
https://github.com/buildpacks/pack/releases/tag/v0.26.0

```suggestion
          version: 0.26.0
```